pkg/configurationManager/configurationManager.go

```diff
@@ -17,6 +19,59 @@ func New() *ConfigurationManager {
 	return &ConfigurationManager{}
 }
 
+func (cm *ConfigurationManager) initConfigFile() error {
+	configHome, configName, configType, err := setViperConfig()
+	if err != nil {
+		return err
+	}
+	// workaround for creating config file when not exist
+	// open issue in viper: https://github.com/spf13/viper/issues/430
+	// should be fixed in pr https://github.com/spf13/viper/pull/936
+	configPath := filepath.Join(configHome, configName+"."+configType)
+
+	isDirExists, err := fileManager.IsExists(configHome)
+	if err != nil {
+		return err
+	}
+	if !isDirExists {
+		osMkdirErr := os.Mkdir(configHome, os.ModePerm)
+		if osMkdirErr != nil {
+			return osMkdirErr
+		}
+	}
+
+	isConfigExists, err := fileManager.IsExists(configPath)
+	if err != nil {
+		return err
+	}
+	if !isConfigExists {
+		_, osCreateErr := os.Create(configPath)
+		if osCreateErr != nil {
+			return osCreateErr
+		}
+	}
+
+	err = viper.ReadInConfig()
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func setViperConfig() (string, string, string, error) {
+	configHome := fileManager.GetAlleroHomedir()
```

Comment:
Is this correspond with the config.json file that we currently have?
We have some functions for reading and writing the file. Please check if there are duplications and let's delete them.